packages/@sanity/cli/src/actions/init-project/readPackageJson.ts

```diff
@@ -0,0 +1,18 @@
+import fs from 'node:fs'
```

Comment:
This exists in https://github.com/sanity-io/sanity/next/packages/sanity/src/_internal/cli/util/readPackageJson.ts, can it be reused?